source/containers/Form/Form.tsx

```diff
@@ -1,5 +1,5 @@
 import PropTypes from 'prop-types';
-import React, { useEffect, useState, useContext } from 'react';
+import React, { useEffect, useState, useContext, createRef, useRef } from 'react';
```

Comment:
`createRef` and `useRef` has been added in this PR, but they don't seem to be used in this file. Is this is a mistake? There only seems to be styling changes in this file.

Reply:
I was working in this file for testing my solution, missed removing the imports.